src/java.base/unix/native/libjli/java_md_common.c

```diff
@@ -204,8 +204,10 @@ JLI_ReportErrorMessageSys(const char* fmt, ...) {
     va_start(vl, fmt);
     vfprintf(stderr, fmt, vl);
 
-    /* TODO: it's safer to use strerror_r, but it isn't available on
-       Solaris 8. Until then.... */
+    /*
+     * TODO: its safer to use strerror_r but is not available on
+     * Solaris 8. Until then....
```

Comment:
We are well past then :) so this should be changed to use strerror_r - though see ./java.base/unix/native/libjava/jni_util_md.c

Reply:
Looks like the names in jni_util_md.c are also hopelessly inaccurate - what would equate to strlen(strerror_r(errno)) and returns a size_t is named getLastErrorString :P

Reply:
Using getLastErrorString in the implementation for Unix, the Windows version of getLastErrorString is also broken, just like JLI_ErrorMessageSys, but unlike the latter shared code does use it, so no attempts will be made to fix it